src/index.html

```diff
@@ -5,8 +5,355 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="font">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list" >
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link" >Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link" >Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link "
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link" >Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link" >Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link" >Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link" >Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card">
+      <div class="card__container">
+        <div class="card__item" data-qa="card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
```

Comment:
Fix formatting everywhere
```suggestion
          <img
            class="card__img"
            src="../src/images/imac.jpeg"
            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
          >
```